packages/pg/lib/stream.js

```diff
@@ -3,12 +3,12 @@
  * @returns {Duplex}
  */
 module.exports.getStream = function getStream(ssl) {
-  const net = require('net')
```

Comment:
The default / first attempt really should be the real socket in node. Trying for `pg-cloudflare` first is weird in that the other 99.99% of usage of this module don't want that code path.

I'd imagine this would do weird things if they happen to have it in their node_modules for other reasons (e.g. they have the dep but are running locally and not on cloudflare). A project could have that module installed or other "cloud node" shim modules for testing against them without actually intending to use any one of them.

Ideally what we need is to have the CF versions of those libraries actually match the node versions (e.g. if CF provides a `net.Socket(...)` it should be usable as a `net.Socket`. Ditto for the TLS stuff. If it just works then none of these shims are required.

Alternatively, come up with a well defined way to signal to the app that it is running on CF. Like checking for an env var.

Reply:
Yeah good call. I think we can use something like:

```ts
process?.release?.name === 'node'
```